### DISCUSSION TITLE
Creation of some source of a migration guide

### DISCUSSION THREAD

**Discussion by rfdzt** [primefaces/community#1203] [PrimeReact]
I like the migration guide that you guys have on primefaces documentation, and is something real useful. I know people with older version of primereact in there projects because they encounter problems when migrating to the latest one and cause there is not some documentation they can follow up to debug there flow when migrating they just give up. And newer version of primereact have really cool features that are just getting missed for those folks.

**Comment by melloware** [comment]
Yeah I tend to agree with you. On the JSF side we are pretty good about tracking migration guide but in the JS libs not so much.    Not making excuses but i think its because the JS world is evolving so fast every day with CSS Variables, SSR, ARIA, Accessibility that the changes are fast and furious.  

JSF we have had the luxury of 15 years of in production so its much easier to track migration changes.  Once again NOT an excuse. 😄
